src/server.conf.in

```diff
@@ -157,10 +157,10 @@ enabled = false
 ;   when allowRegistration is true. While certificate mode offers the highest
 ;   security it is currently considered experimental and the user experience
 ;   varies between browsers and platforms.
-;mode = sharedsecret
+mode = sharedsecret
 ; The shared secred for HMAC validation in "sharedsecret" mode. Best use 32 or
 ; 64 bytes of random data.
-;sharedsecret_secret = some-secret-do-not-keep
+sharedsecret_secret = some-secret-do-not-keep
```

Comment:
How is `sharedsecret_secret` changed? I do not see it in the config hook. I guess we do not want to use that value hardcoded. Is there a way so the config hook could read this value from Nextcloud somehow?